tests/unit/test_exec.py

```diff
@@ -1,9 +1,95 @@
 __copyright__ = "Copyright (c) 2020-2021 Jina AI Limited. All rights reserved."
 __license__ = "Apache-2.0"
 
+import numpy as np
+
+from jina import Document, DocumentArray
 from jina.executors import BaseExecutor
+from jinahub.segmenter.vggish_audio_segmenter import VGGishSegmenter
 
 
 def test_exec():
     ex = BaseExecutor.load_config('../../config.yml')
-    assert ex._dim == 128
+    assert ex.chunk_duration == 10
+
+
+def test_mono():
+    n_frames = 100
+    frame_length = 2048 # the number of samples in each frame
+    signal_orig = np.random.randn(frame_length * n_frames)
+
+    segmenter = VGGishSegmenter()
+    docs = DocumentArray([Document(blob=signal_orig,
+                                   tags={'n_frames': 100, 'frame_length': 2048, 'sampling_rate': 44100})])
+    segmenter.segment(docs, {})
+    assert len(docs) == 1
+
+    chunks = docs.traverse_flat(['c'])
+    assert len(chunks) == 0
+
+
+def test_stereo():
+    n_frames = 100
+    frame_length = 2048
+    signal_orig = np.random.randn(2, frame_length * n_frames)
+
+    segmenter = VGGishSegmenter()
+    docs = DocumentArray([Document(blob=np.stack([signal_orig, signal_orig]),
```

Comment:
this is incorrect. It should be `Document(blob=signal_orig,...` without the stack